File: 02-manipulaArquivos/python/aluno.py
class Aluno:
    #metodo construtor
    def __init__(self, nome, idade):
        self.nome = nome
        self.idade = idade
        self.gerar_email()

    def gerar_email(self):
        vetor_palavras = self.nome.split(" ")
        if (len(vetor_palavras) > 1):
            self.email = vetor_palavras[0].lower() + "." + vetor_palavras[1].lower() + "@ufn.edu.br"
        else:
            self.email = vetor_palavras[0].lower() + "@ufn.edu.br"
    
    #override do toString
    def __str__(self):
        return self.nome + " - " + self.email
    

File: 02-manipulaArquivos/C#/ConsoleLeituraArquivo/Aluno.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace ConsoleLeituraArquivo
{
    internal class Aluno
    {
        public string Nome { get; set; }
        public int Idade { get; set; }
        public string Email { get; set; }

        public Aluno(string nome, int idade)
        {
            Nome = nome;
            Idade = idade;
            this.gerarEmail();
        }

        private void gerarEmail()
        {
            string[] vetorPalavras = Nome.Split(" ");
            //[joao, victor]
            if (vetorPalavras.Length > 1)
            {
                Email = vetorPalavras[0] + "." + vetorPalavras[1] + "@ufn.edu.br";
            }
            else
            {
                Email = vetorPalavras[0] + "@ufn.edu.br";
            }
        }
        
        public override string? ToString()
        {
            return Nome + " - " + Email;
        }
    }
}
